### DISCUSSION TITLE
focusRing: "always" will provide styles for components that should not be activated by buttons

### DISCUSSION THREAD

**Discussion by Mangomilkhh** [mantinedev/mantine#3580] [Q&A] [answered]
The attribute theme of the MantineProvider component is set to focusRing: "always", which will cause the DateRangePicker component to automatically activate the button style. Each time you click on it, you will give the 1st date focus style.
As soon as the Drawer component is opened, it will automatically give the focus style to the close button.

**Comment by Mangomilkhh** [comment]
Effect of opening for the first time.
![1676899878686](https://user-images.githubusercontent.com/86409087/220122714-46841b22-7174-4ebf-b851-f9c55bfca1bb.jpg)

![1676899919829](https://user-images.githubusercontent.com/86409087/220122796-0433ffbf-6d96-4b9b-97fe-a4818109c602.jpg)

**Comment by rtivital** [comment]
Please use English to provide feedback, maintainers do not speak other languages

**Comment by rtivital** [comment, marked as answer]
It is correct behavior. Drawer, DateRangePicker and other similar components use soft focus trap. In order for it to work (and also make component accessible) first interactive element inside must be focused. You can customize initial focused element with `data-autofocus` attribute – https://mantine.dev/core/modal/#initial-focus

**Reply by Mangomilkhh** [reply]
Thank you. It has been solved.
